testutils/clustermanager/util_test.go

```diff
@@ -17,29 +17,33 @@ limitations under the License.
 package clustermanager
 
 import (
-	"fmt"
-	"reflect"
 	"testing"
+
+	"knative.dev/pkg/testutils/common"
 )
 
-func TestStandardExec(t *testing.T) {
-	datas := []struct {
-		cmd    string
-		args   []string
-		expOut string
-		expErr error
-	}{
-		{"bash", []string{"-c", "echo foo"}, "foo\n", nil},
-		{"cmd_not_exist", []string{"-c", "echo"}, "", fmt.Errorf("exec: \"cmd_not_exist\": executable file not found in $PATH")},
+func TestGetResourceName(t *testing.T) {
+	buildNumStr := "12345678901234567890fakebuildnum"
+	expOut := "kpkg-e2e-cls"
+	if common.IsProw() {
```

Comment:
While your code is ready for both cases (Prow/local) you're only testing one of them. You should test both cases.

Reply:
done